src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 * Upgraded Azure.Core to 1.44.1.
+* Added Vector Indexing properties to Sql container APIs.
```

Comment:
Move the 3 lines to right after "Upcoming Release".
For each line, start with the past tense of the verb, like "Added", "Updated", ...